.github/actions/destroy-cluster/action.yaml

```diff
@@ -16,15 +16,18 @@ inputs:
   # Tooling configuration
   TERRAFORM_VERSION:
     description: "Terraform version to use"
-    required: true
+    required: false
+    default: "1.3.6"
```

Comment:
A single source of truth in (today) 5 different workflows ? 

To me this version shouldn't be configurable, it's hardcoded somewhere and that's it
With this PR, it's only in 2 different actions (we could indeed move it to a single place, I don't think a github variable is a good idea because we may change this value depending on the branch)